docs/content/python/getting-started/quickstart/index.md

```diff
@@ -87,59 +85,29 @@ resource(_type: Page, "page", actions, roles) if
             permissions: ["read"]
         },
         admin: {
-            permissions: ["write"],
-            implies: ["guest"]
+            permissions: ["write", "read"]
         }
     };
  ```
 
-The `allow` rule is the top-level rule that we use to say who can do what in our application.
-In this case, we are delegating to Oso's built-in `role_allows` rule which implements all the
-authorization logic for role-based access control based on the data we provide in `actor_has_role_for_resource`
-and `resource`.
-
-An `actor_has_role_for_resource` rule looks up role objects that are associated with an actor.
-Role objects are of the form `{name: "the-role-name", resource: TheResourceObject}`.
-The Oso builtin roles will look up this rule, so this is required.
+The `allow` rule is what's queried by `is_allowed()` in your application.
+In this case, we are delegating to Oso's built-in `role_allows` rule.
```

Comment:
I might just say `calling` instead of `delegating`. More straightforward terminology.